src/PhysicsEngine.cpp

```diff
@@ -217,6 +217,9 @@ void RungeKutta::march_step(double tStamp, double tStep) {
 
     double mass = rocket_.get_mass();
 
+    Vector3d geod = rocket_.get_r_geod();
+    Vector3d ecef = rocket_.get_r_ecef();
```

Comment:
geod and ecef are not really used. ecef is just used to set ecef at the end of the function and geod is never used